application/templates/cms/edit_measure_page.html

```diff
@@ -266,6 +266,11 @@ <h3 class="heading-medium">Dimensions</h3>
                         {% for dimension in measure.dimensions %}
                             <tr class="movable" data-dimension-guid="{{ dimension.guid }}">
                                 <td>{{ dimension.title }}</td>
+                                <td>
+                                    {% if dimension.page.version != '1.0' and dimension.updated_at != dimension.page.updated_at %}
+                                        <span class="updated">Updated</span>
```

Comment:
The Design System pattern seems to have this as all caps. Do we want to copy them there?

Reply:
@samuelhwilliams I considered that too. But I'm also told that ALL CAPS can be bad for accessibility, so I shied away from it.

Reply:
Interesting. Maybe worth feeding that back/asking about it on the 'task list' PR in design system?